src/components/BottomSheet.tsx

```diff
@@ -89,8 +95,8 @@ const styles = StyleSheet.create({
   },
   background: {
     position: 'absolute',
-    backgroundColor: colors.modalBackdrop,
-    opacity: 0.5,
+    backgroundColor: '#2E3338',
+    opacity: 0.25,
```

Comment:
are these safe to change? These are used in places other than onboarding right?

Reply:
probably best not to. I'll refactor